.sonarcloud.properties

```diff
@@ -0,0 +1,7 @@
+# Path to sources
+sonar.sources=.
+sonar.exclusions=**/*test*, **/vendor/**, **/zz_generated.*, **/*.yaml,
```

Comment:
`**/*test*,` will catch files like https://github.com/redhat-appstudio/internal-services/blob/main/controllers/internalrequest/adapter_test.go won't it? Shouldn't we still run sonarcloud against our test files? I don't know much about sonarcloud - does it have value to run against ginkgo test files?

Reply:
we can run it against test files but test file would have a lot of duplicates that is why i tend to exclude test files 

for example you can see all of the duplicates right now are encountered in test files https://sonarcloud.io/component_measures?id=redhat-appstudio_internal-services&metric=duplicated_lines&view=list